### PR TITLE
Mark platform as "machine only" for rules which are missing and are required to have

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Mark rules required to have platform as machine only.

#### Rationale:

- Some rules are not applicable in container scanning, therefore they need to be marked as machine only. Some rules appear to have it as unmarked, but its group was marked as machine instead so it will inherit this parameter.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/rule.yml [hunk 1]
```diff
@@ -33,3 +33,5 @@ references:
 ocil_clause: 'the pcscd service is not enabled'
 
 ocil: '{{{ ocil_service_enabled(service="pcscd") }}}'
+
+platform: machine
```
Comment (yuumasato) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/rule.yml]: Sorry, I didn't mean to have a blank newline, just to have a newline at the end of the line.
Check: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/rule.yml]: See the marker "no newline at end of file'

Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/rule.yml]: Which can only be seen in the "Files changed" tabs, oh, GitHub.

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/rule.yml]: Done

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml [hunk 2]
```diff
@@ -11,3 +11,5 @@ description: |-
     In Red Hat Enterprise Linux servers and workstations, hardware token login
     {{% endif %}}
     is not enabled by default and must be enabled in the system settings.
+
+platform: machine
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml]: NACK. If someone is able to login, that logon should be CAC/PIV enabled. Rules like this should not have ``platform: machine``

File: linux_os/guide/system/software/integrity/software-integrity/aide/aide_use_fips_hashes/rule.yml [hunk 3]
```diff
@@ -41,3 +41,5 @@ ocil: |-
     To determine that AIDE is configured for FIPS 140-2 file hashing, run the following command:
     <pre>$ grep sha512 /etc/aide.conf</pre>
     Verify that the <tt>sha512</tt> option is added to the correct ruleset.
+
+platform: machine
```
Comment (shawndwells) [inline linux_os/guide/system/software/integrity/software-integrity/aide/aide_use_fips_hashes/rule.yml]: NACK. Containers and VMS are still expected to perform integrity checking.

**Comment by shawndwells** [comment]
What's the best way to handle? Should we revert the PR until changes are made?

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml (hunk 2 above)
Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml]: It seems like an overkill to me, but if people wanna do it, we shall not prevent them.

**Comment by yuumasato** [comment]
> What's the best way to handle? Should we revert the PR until changes are made?

Probably a new PR.

**Comment by ggbecker** [comment]
New PR to clarify which rules are applicable to containers and address issues raised in this PR: https://github.com/ComplianceAsCode/content/pull/4246

**Pull request merged**